pkgs/by-name/pd/pdk/package.nix

```diff
@@ -0,0 +1,23 @@
+{ lib, bundlerApp, bundlerUpdateScript, makeWrapper, gnumake }:
+
+bundlerApp {
+  pname = "pdk";
+  gemdir = ./.;
+  exes = [ "pdk" ];
+
+  nativeBuildInputs = [ makeWrapper ];
+
+  postBuild = ''
+    wrapProgram $out/bin/pdk --prefix PATH : ${lib.makeBinPath [ gnumake ]}
+  '';
```

Comment:
why are you wrapping the program with gnumake?

Reply:
Because it calls make for some steps, e.g. at `pdk new module [name]`.